app/views/welcome/index.html.erb

```diff
@@ -7,64 +7,18 @@
   <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
 </head>
 
+<div>
+  <% @categories.each do |category|  %>
+    <%= link_to category.name, category_path(category)  %>
+  <% end %>
+</div>
 
-<div class="welcome">
-<div class="carouselcontainer">
-  <strong><h3 id='carouseltitle'>REAL SPECIAL ITEMS</h3>
-  <div id="myCarousel" class="carousel slide" data-ride="carousel">
-    <!-- Indicators -->
-    <ol class="carousel-indicators">
-      <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
-      <li data-target="#myCarousel" data-slide-to="1"></li>
-      <li data-target="#myCarousel" data-slide-to="2"></li>
-    </ol>
-    <% if @item3 %>
-    <!-- Wrapper for slides -->
-    <div class="carousel-inner">
-
-      <div class="item active">
-        <%= image_tag @item1.image_path, class: "carouselimg"%>
-        <div class="carousel-caption">
-          <h4> <%= @item1.name %> </h4>
-          <p> <%= @item1.description %>  <%= button_to "info_outline", item_path(@item1.id), class: "btn-large black large material-icons" , method: 'get' %> </p>
-        </div>
-      </div>
-
-      <div class="item">
-          <%= image_tag @item2.image_path, class: "carouselimg" %>
-          <div class="carousel-caption">
-            <h4> <%= @item2.name %> </h4>
-            <p> <%= @item2.description %>          <%= button_to "info_outline", item_path(@item2.id), class: "btn-large black large material-icons" , method: 'get' %></p>
-          </div>
-      </div>
-
-      <div class="item">
-          <%= image_tag @item3.image_path, class: "carouselimg" %>
-          <div class="carousel-caption">
-            <h4> <%= @item3.name %> </h4>
-            <p> <%= @item3.description %>  <%= button_to "info_outline", item_path(@item3.id), class: "btn-large black large material-icons" , method: 'get' %></p>
-          </div>
-      </div>
-
-      <% end %>
-
-    </div>
-
-    <!-- Left and right controls -->
-    <a class="left carousel-control" href="#myCarousel" data-slide="prev">
-      <span class="glyphicon glyphicon-chevron-left"></span>
-      <span class="sr-only">Previous</span>
-    </a>
-    <a class="right carousel-control" href="#myCarousel" data-slide="next">
-      <span class="glyphicon glyphicon-chevron-right"></span>
-      <span class="sr-only">Next</span>
-    </a>
-  </div>
+<div>
+  <% @stores.each do |store| %>
+    <%= link_to store.name, store_path(store) %>
+  <% end %>
 </div>
-<br />
-<br />
-<br />
 
-<%= image_tag "bury.gif" %>
-<%= image_tag "trash.gif" %>
+<div>
+  <%= link_to "View Stores", stores_path %>
 </div>
```

Comment:
Why are we taking out the carousel? I thought we were going to have a carousel of items?